nmostesting/suites/IS1101Test.py

```diff
@@ -211,49 +210,47 @@ def test_02_00(self, test):
                     self.compat_url + "senders/" + sender + "constraints/active/",
                 )
                 if response.status_code != 200:
-                    return test.FAIL("senders constraints cannot be deleted")
+                    return test.FAIL("The sender {} constraints cannot be deleted".format(sender))
             return test.PASS()
-        return test.UNCLEAR("There is no IS-11 senders.")
+        return test.UNCLEAR("There is no IS-11 senders")
```

Comment:
Consistent singular or plural?

Reply:
I think this is accurate. In the first FAIL message we are talking about a single 'sender' and in the UNCLEAR message we are talking about them all so 'senders'.

Reply:
I wonder if this is a French/English difference!